src/ui/TypingIndicatorMessage/index.tsx

```diff
@@ -1,44 +1,80 @@
-import React, { ReactElement, useContext } from 'react';
+import React from 'react';
 import { Member } from '@sendbird/chat/groupChannel';
-import { LocalizationContext } from '../../lib/LocalizationContext';
-import Avatar from '../Avatar';
-import Label, { LabelColors, LabelTypography } from '../Label';
+import Avatar, {DefaultComponent} from '../Avatar';
 import TypingDots from './TypingDots';
 
 export interface TypingIndicatorMessageProps {
-  typingMember: Member;
+  typingMembers: Member[];
 }
 
-export default function TypingIndicatorMessage({
-  typingMember,
-}: TypingIndicatorMessageProps): ReactElement {
-  const { stringSet } = useContext(LocalizationContext);
+const AVATAR_BORDER_SIZE = 2;
+const AVATAR_DIAMETER = 26 + AVATAR_BORDER_SIZE;
+const LEFT_GAP = 20;
+
+const TypingIndicatorMessageAvatar = (props : React.PropsWithChildren<TypingIndicatorMessageProps>) => {
```

Comment:
You can remove `React.PropsWithChildren<...>` if you don't need a `children` prop.